### PR TITLE
Add release notes for 19.3

### PR DESCRIPTION
**Pull request by ketan**

### CODE REVIEW

File: source/partials/next_release/_release.md.erb [hunk 1]
```diff
@@ -1,102 +1,54 @@
...
-* <%= link_to_issue 5705, 'Fix for high CPU usage on console logs/tests page.' %>
-* <%= link_to_pull_request 5897, 'Killing agent via stop command honors the process ID' %>
-* <%= link_to_issue 5788, 'Fixed back button navigation on dashboard.' %>
+* <%= link_to_issue 5956, 'Fixed issue with environments API incorrectly rendering encrypted variables as `******` instead of the correct encrypted value.' %>
+* <%= link_to_issue 5886, 'Do not allow assigning an agent to an environment defined in config repository environment.' %>
+* <%= link_to_issue 5968, 'Fix spurious `e.map is not a function` javascript error for notifications.' %>
+* <%= link_to_pull_request 5974, "Unable to expand/collapse environments with name containing special characters." %>
+* <%= link_to_issue 5962, "GoCD shows HTTP Basic Auth dialog when after user is logged out." %>
+* <%= link_to_pull_request 6011, "Fix errors with parsing git log when coloring is turned on using `.gitconfig`." %>
+* <%= link_to_pull_request 5967, "Environments api should accept both `encrypted_value` and `value`." %>
```
Comment (kritika-singh3) [inline source/partials/next_release/_release.md.erb]: Does it give an impression that the user can provide both the properties at the same time?

File: source/partials/next_release/_release.md.erb [hunk 2]
```diff
@@ -1,102 +1,54 @@
...
-* The Fedora 28 based docker image for gocd agent has been deprecated and will not be built after June 2019 release of GoCD. This will be done as Fedora 28 will reach end-of-life in June 2019.
-* The Dashboard API v2 has been deprecated. This version of the API will be removed in a release scheduled for June 2019. Version 3 of the API is available, and users are encouraged to use it.
 
+* elastic extension v4
+* elastic agent profile v1 (breaking change)
 
 <h4>Deprecations</h4>
-* The elastic agent plugin extension version 3 has been deprecated. This version will be removed in a release scheduled for Mar 2019. Plugin developers should use version 4 of the elastic agent plugin api to allow adding support for job completion request. The docker, docker swarm, kubernetes and ecs plugins have been migrated to the new versions, and users should upgrade their plugins to the latest version to see these new features.
+
 * The notification plugin extension version 3 has been deprecated. This version will be removed in a release scheduled for Mar 2019. Plugin developers should use version 4 of the notification plugin api.
```
Comment (kritika-singh3) [inline source/partials/next_release/_release.md.erb]: We may have to think about a new timeline to remove this or we have to migrate all gocd supported notification plugins. /cc @arvindsv @maheshp 

The same goes for the analytics plugin.

File: source/partials/next_release/_release.md.erb [hunk 3]
```diff
@@ -1,102 +1,54 @@
...
 * The analytics plugin extension version 1 has been deprecated. This version will be removed in a release scheduled for Mar 2019. Plugin developers should use version 2 of the analytics plugin api.
 * The Get Server Info processor version 1 has been deprecated. This version will be removed in a release scheduled for April 2019. Plugin developers should use version 2 of the Get Server Info processor.
```
Comment (kritika-singh3) [inline source/partials/next_release/_release.md.erb]: I am happy to submit PR for the same :)

File: source/partials/next_release/_release.md.erb [hunk 4]
```diff
@@ -107,36 +59,36 @@ We have rewritten the User summary page, now known as User Management, to improv
   "Aravind SV",
   "Bhupendrakumar Piprava",
   "Chandrakanth Nagaraj",
+  "Dhanashri Pagar",
```
Comment (kritika-singh3) [inline source/partials/next_release/_release.md.erb]: Need to add  Ketan Sangle, Jyoti Singh, Sandro Heinzelmann in contributors list

File: source/partials/next_release/_release.md.erb (hunk 2 above)
Comment (arvindsv) [reply source/partials/next_release/_release.md.erb]: @kritika-singh3 Agreed. Let's not remove it this time.

File: source/partials/next_release/_release.md.erb (hunk 3 above)
Comment (arvindsv) [reply source/partials/next_release/_release.md.erb]: :) It wouldn't be bad for more people to take a look at the plugins. Feel free to do that, but we don't need to rush it, at this time, for this release. I think. /cc @maheshp

Comment (maheshp) [reply source/partials/next_release/_release.md.erb]: Agreed, we can push it to the next release. @kritika-singh3 do you have a list of plugins which needs to be migrated?

Comment (kritika-singh3) [reply source/partials/next_release/_release.md.erb]: Notification plugin

| Plugin Name                          | Repo                                       | Version |
| ------------------------------------ | ------------------------------------------ | ------- |
| AWS SNS Notifier                     | paullalonde/gocd-sns-notification-plugin   | v1.0    |
| Build-Watcher Slack&Email Notifier   | gmazzo/gocd-build-watcher-plugin           | v1.0    |
| Slack Build Notifier                 | ashwanthkumar/gocd-slack-build-notifier    | v1.0    |
| WebSockets Build Notifier            | matt-richardson/gocd-websocket-notifier    | v1.0    |
| HipChat Notification                 | PagerDuty/gocd-hipchat-plugin              |   NA      |
| Riemann Notification                 | rsr5/gocd-riemann-notifier                 | v1.0    |
| Spark Build Notifier                 | t04glovern/gocd-spark-build-notifier(fork) | v1.0    |
| Rocket.Chat Build Notifier           | tomzo/gocd-rocketchat-plugin               | v3.0    |
| MicroFocus ALM Octane GoCD CI Plugin | MicroFocus/octane-gocd-plugin              | v1.0    |
| Email Notifier                       | gocd-contrib/email-notifier                | v1.0    |
| GitHub Pull Request Status Notifier  | gocd-contrib/gocd-build-status-notifier    | v1.0    |
| Stash Pull Request Status Notifier   | gocd-contrib/gocd-build-status-notifier    | v1.0    |
| Gerrit Change Set Status Notifier    | gocd-contrib/gocd-build-status-notifier    | v1.0    |
| Gitter Notification                  | gocd-contrib/gitter-notifier               | v1.0    |
| Gitter Activity Feed                 | gocd-contrib/gitter-activity-feed-plugin   | v1.0    |

Analytics plugin

| Plugin Name                          | Repo                               | Version |
| ------------------------------------ | ---------------------------------- | ------- |
| Analytics Plugin (Paid)              | gocd-private/gocd-analytics-plugin | v1.0    |

**Pull request merged**